### PR TITLE
docs: update content and file list

### PR DESCRIPTION
**Pull request by pocketdr**
### Description

Improving machine translation and updating file list

### CODE REVIEW

File: files/zh-cn/web/accessibility/index.md [hunk 1]
```diff
@@ -7,47 +7,47 @@ slug: Web/Accessibility
...
-
-- [自动化无障碍测试](http://www-archive.mozilla.org/quality/embed/plans/accessibility/nsIAccessibleTestPlan.html)
-- [尖牙屏幕阅读模拟器](http://www.standards-schmandards.com/index.php?show/fangs)
-
-## 其他有用的网站
-
-- [屏幕阅读器列表](https://support.mozilla.org/kb/accessibility-features-firefox-make-firefox-and-we)
+  - : 学习如何利用无障碍富网络应用（Accessible Rich Internet Application）使你的 HTML 文档进一步无障碍化的一组文章。
+- [移动无障碍化清单](/zh-CN/docs/Web/Accessibility/Mobile_accessibility_checklist)
+  - : 本文档向移动应用开发者提供了实现无障碍化所需内容的简明清单
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/index.md:44]:
```suggestion
  - : 本文档向移动应用开发者提供了实现无障碍化所需内容的简明清单。
```

File: files/zh-cn/web/accessibility/index.md [hunk 2]
```diff
@@ -7,47 +7,47 @@ slug: Web/Accessibility
...
+- [认知无障碍](/zh-CN/docs/Web/Accessibility/Cognitive_accessibility)
+  - : 本文说明了如何确保你的网站内容对认知障碍群体是无障碍的。
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/index.md:46]:
```suggestion
  - : 本文说明了如何确保你创造的 web 内容对认知障碍群体是无障碍的。
```

File: files/zh-cn/web/accessibility/index.md [hunk 3]
```diff
@@ -7,47 +7,47 @@ slug: Web/Accessibility
...
+- [对癫痫症无障碍](/zh-CN/docs/Web/Accessibility/Seizure_disorders)
+  - : 对于伴有特定脑部缺陷的群体，网络上的某些视觉内容可能导致他们出现癫痫症状。本文可以帮助你理解哪一类内容会有这些问题，并提供了避开这些问题的策略与工具。
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/index.md:48]:
```suggestion
- [对癫痫症患者的无障碍](/zh-CN/docs/Web/Accessibility/Seizure_disorders)
  - : 对于伴有特定脑部缺陷的群体，web 上的某些视觉内容可能导致他们出现癫痫症状。本文可以帮助你理解哪一类内容会有这些问题，并提供了避开这些问题的策略与工具。
```

File: files/zh-cn/web/accessibility/index.md [hunk 4]
```diff
@@ -7,47 +7,47 @@ slug: Web/Accessibility
...
+
+## 参见
+
+- [WAI 利益团体](https://www.w3.org/WAI/about/groups/waiig/)
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/index.md:52]: https://www.w3.org/2020/08/member-guidebook-zh.html

```suggestion
- [WAI 兴趣组](https://www.w3.org/WAI/about/groups/waiig/)
```

**Review by yin1999 — COMMENTED** [review]
some suggestions

File: files/zh-cn/web/accessibility/index.md [hunk 5]
```diff
@@ -7,47 +7,47 @@ slug: Web/Accessibility
...
+- [多媒体无障碍化](/zh-CN/docs/Learn/Accessibility/Multimedia)
+  - : 多媒体也是一类可能需要无障碍化的内容——视频、音频、图像等都需要给出适当的文本替代，以便可以被辅助性技术及其用户来理解。本文就介绍了以上内容。
+- [移动设备的无障碍](/zh-CN/docs/Learn/Accessibility/Mobile)
+  - : 使用移动设备上网越来越流行，而同时，iOS、Android 等主流移动平台也拥有日趋完善的辅助性工具，让你的网站内容在这些平台上做到无障碍化自然很重要。本文将介绍移动设备领域的无障碍化注意事项。
 
-## 其他文件
+## 其他文章
 
 - [了解 Web 内容无障碍功能指南](/zh-CN/docs/Web/Accessibility/Understanding_WCAG)
   - : 这组文章提供了快速解释，以帮助你了解需要采取的步骤，以符合 W3C web 内容辅助功能指南 2.0 的建议
```
[the remark this answers is not in the document]
Comment (pocketdr) [reply files/zh-cn/web/accessibility/index.md:36]: 漏掉了（

File: files/zh-cn/web/accessibility/index.md [hunk 6]
```diff
@@ -7,47 +7,47 @@ slug: Web/Accessibility
   {{ListSubpagesForSidebar("/zh-CN/docs/Web/Accessibility", 1)}}
 </section>
 
-在 Web 开发无障碍性意味着使尽可能多的人能够使用 Web 站点，即使这些人的能力是有限的。这里我们提供关于开发易访问的内容的信息。
+**无障碍**（Accessibility，常被缩写为 **A11y**，因为它以“a”开头，后跟 11 个字母，以“y”结尾）在 web 开发中意味着使尽可能多的人能够使用网站，即使有些用户的能力在某种程度上受限。
 
-“无障碍性是最常用于描述设施或设施，帮助残疾人，如“轮椅”。这可以扩展到盲文标识、轮椅坡道，音频信号在人行横道，轮廓人行道，网站设计，等等。" [维基百科条目无障碍](http://en.wikipedia.org/wiki/Accessibility)
+对大多数人来说，技术让生活更便捷。而对于有障碍的人群，技术让本不可能的事变为可能。无障碍化意味着设计出尽可能没有使用障碍的内容，无论访问者的身体情况、认知能力以及他们访问网络的方式是怎样的。
```
Comment (yin1999) [inline files/zh-cn/web/accessibility/index.md:12]:
```suggestion
对大多数人来说，技术让生活更便捷。而对于有障碍的人群，技术使一切皆有可能。无障碍化意味着设计出尽可能没有使用障碍的内容，无论访问者的身体情况、认知能力以及他们访问 web 的方式是怎样的。
```

**Review by yin1999 — COMMENTED** [review]
One more suggestion :)

File: files/zh-cn/web/accessibility/index.md (hunk 6 above)
Comment (pocketdr) [reply files/zh-cn/web/accessibility/index.md:12]: thx

**Review by yin1999 — APPROVED** [review]
Thank you @pocketdr!

**Pull request merged**